test/unit/int_algolia/scripts/algolia/lib/algoliaSplitter.test.js

```diff
@@ -36,6 +36,24 @@ describe('HTML Content Manipulation', () => {
             const result = splitHtmlContent(htmlContent, maxByteSize, splitterElement);
             expect(result).toEqual(['Bold Text and normal text']);
         });
+
+        test('should remove self-closing ignored tags', () => {
+            const htmlContent = '<div>Content with image<img src="image.jpg" alt="Image"> more content</div>';
```

Comment:
You wanted a closing tag for this one, didn't you?
```suggestion
            const htmlContent = '<div>Content with image<img src="image.jpg" alt="Image"/> more content</div>';
```